modules/pattern/pattern.resolver.js

```diff
@@ -18,7 +20,18 @@ const patternQuery = {
 const patternMutation = {
   addPattern: async (parent, args) => {
     try {
-      return await patternService.addPattern(args.pattern);
+      if (!args.upload) {
```

Comment:
I don't think that this can be true, pattern without image doesn't have a lot of sense, its image should be required argument

Reply:
fixed